src/main/java/com/gtnewhorizons/modularui/api/screen/ModularUIContext.java

```diff
@@ -283,6 +289,13 @@ public boolean tryApplyStoredPos(ModularWindow window) {
         return false;
     }
 
+    public void storeWindowPos(int id) {
```

Comment:
Imo this logic can be merged to `storeWindowPos` and `tryApplyStoredPos`, so that window pos will be stored without explicitly calling the method. You can retrieve window id with `syncedWindows.inverse()`.